src/requirements/types.ts

```diff
@@ -90,6 +90,7 @@ type GenericRequirementsJson<R> = {
   readonly university: UniversityRequirements;
   readonly college: CollegeRequirements<R>;
   readonly major: MajorRequirements<R>;
+  readonly minor: MajorRequirements<R>;
```

Comment:
Same with this?

Reply:
We could investigate a little bit more, but I believe it's the same case

Reply:
Made a Monday item for it